src/pyspextool/extract/find_peaks.py

```diff
@@ -131,8 +131,8 @@ def find_peaks(profiles, method_info, fwhm=0.8):
 
         for i in range(norders):
 
-            x = profiles[i]['y']
-            y = profiles[i]['p']
```

Comment:
I don't see changes to make_profiles in the request so did you change the output of that program from 'y' and 'p' to 'angle' and 'profile'?

Reply:
Something is weird here as my version (and the main branch here) has some of these changes to find_peaks.py.  I am just confused as to why they are coming up again in a your pull request. @kelle .